site2/docs/client-libraries-python.md

```diff
@@ -44,23 +36,10 @@ pip install 'pulsar-client[all]==@pulsar:version_number@'
 
 Installation via PyPi is available for the following Python versions:
 
-Platform | Supported Python versions
-:--------|:-------------------------
-MacOS >= 11.0 | 3.7, 3.8, 3.9 and 3.10
-Linux (including Alpine Linux) | 3.7, 3.8, 3.9 and 3.10
-
-
-### Install from source
-
-To install the `pulsar-client` library by building from source, follow [instructions](client-libraries-cpp.md#compilation) and compile the Pulsar C++ client library. That builds the Python binding for the library.
-
-To install the built Python bindings:
-
-```shell
-git clone https://github.com/apache/pulsar
-cd pulsar/pulsar-client-cpp/python
-sudo python setup.py install
-```
```

Comment:
This is simply incorrect. We should update the doc in README.md under `pulsar-client-cpp` and add back this section to guide users there.

Reply:
It's really complex to build the Python client "correctly" from source right now. See https://github.com/apache/pulsar/issues/17468.